### PR TITLE
Explain why consoles/robots are still working [WIP]

### PR DESCRIPTION
**Pull request by i2amroy**
Consoles drop a few "tiny plutonium cell" items, which can be combined at a 10:1 ratio with a little soldering work to form a normal plutonium cell. The average exchange rate is 2-4 disassembled consoles and 2 soldering iron/toolset charges to a single normal plutonium cell. (Sadly because plutonium cells come in batches of 5 you can't just do a 10->1 recipe, we have to do a 50->5 recipe). This lets us explain why there are a bunch of computer consoles still working when we don't have working power; they're running off tiny amounts of nuclear power (which we already do in cases like deep-space satellites or rovers).

Does a slight tweak to leaking mechanics. Count by charges items that would leak when damaged (which is currently just the plutonium cells and the fixed items listed later) now leak by the item rather than by the stack. This means that a stack of 1000 damaged plutonium cells will now leak 1000 radiation, rather than just 1, and you can no longer make damaged cells safer to carry by stacking them.

Also fixed a few occurrences of the old "LEAK_RAD" flag, which has long since been replaced by the "LEAK_DAM"+"RADIOACTIVE" combination.

### CODE REVIEW

**Comment by karlnp** [comment]
:D

**Comment by Petethegoat** [comment]
This does raise the question of why everything doesn't use these cells.

**Comment by i2amroy** [comment]
Actually looking into that right now, I think I'm going to put a few more small changes onto this for robots. :P

**Comment by dopeghoti** [comment]
Does this tie into the "deactivate external power" option in the consoles at evac shelters?

**Comment by i2amroy** [comment]
Wasn't planning to, no. Looking at the description for that mission I see no particular reason why that mission and this would be connected. This is more about computer consoles and mobile robots specifically, that seems aimed more at the grid-like aspect of power at the shelters.

File: src/item.cpp [hunk 1]
```diff
@@ -1625,7 +1625,7 @@ nc_color item::color_in_inventory() const
         ret = c_cyan;
     } else if(has_flag("LITCIG")) {
         ret = c_red;
-    } else if ( has_flag("LEAK_DAM") && has_flag("RADIOACTIVE") && damage > 0 ) {
+    } else if ( (has_flag("LEAK_DAM") || has_flag("LEAK_DAM_SMALL")) && has_flag("RADIOACTIVE") && damage > 0 ) {
```
Comment (BevapDin) [inline src/item.cpp]: I don't like to be that person, but:

The flag appears to be new. Where is it documented?

Comment (i2amroy) [inline src/item.cpp]: Nowhere you can see yet, though it's present on my local copy that I'm still tweaking.

**Comment by i2amroy** [comment]
Since I've since realized that the scope of this PR is slightly bigger than I originally intended (and it looks like there is at least some approval for the widespread ideas) I'm going to close this for now while I hash out some of the more basic ideas that it was originally aiming at. I'll reopen this once I get all that sorted out and it's a little closer to being ready in interest of not having another WIP PR that isn't ready for comments cluttering up the list.

**Pull request closed without merging**